vignettes/migration_from_the_old_crmPack.Rmd

```diff
@@ -21,7 +22,283 @@ The following vignette describes the changes that were introduced to the `crmPac
+# and computes the corresponding dose, according to the model chosen and given a fixed values
+# of the model's parameters.
+true_dose_fun <- doseFunction(my_model, alpha0 = -0.8, alpha1 = 1)
+true_dose_fun(0.3)
+```
+
+## Prob
+
+```{r prob_function}
+# Toxicity probabilities for a given dose (equal to 10), model and samples.
+# Every single probability value corresponds to one particular sample in `my_samples`.
+prob(10, my_model, my_samples)
+
+# True DLT-dose relationship.
+# Say that -0.8 and 1 are the true values for models parameters alpha0 and alpha1 respectively.
+# The `true_prob_fun` takes one argument (the dose) and computes the corresponding
+# toxicity probability, according to the model chosen and given a fixed values
+# of the model's parameters.
+true_prob_fun <- probFunction(my_model, alpha0 = -0.8, alpha1 = 1)
+true_prob_fun(10)
+```
+
+# New Random Number Generator settings for the MCMC
+
+The Random Number Generator (RNG) settings used by the JAGS for the MCMC are now
+configured solely through the `McmcOptions` class. The RNG settings are:
+`RNG type` and the `RNG seed` that corresponds to a given `RNG type`. Find out
+details in the help page for the `McmcOptions` class.
+Any RNG-related user settings at the R session level (such us those with
+`set.seed()`) are ignored by the MCMC sampler.
+
+# New no-argument constructors
+
+To aid software development, new no-argument constructs for all sub-classes of `GeneralModel`, `Increments`, `NextBest` and `Stopping` have been introduced.  The names of these constructors take the form `.Default<classname>`, where `<classname>` is the name of the class being created.
+
+These constructors return valid, but not necessarily contextually sensible, objects of the required class.  One reason the objects returned may not be contextually sensible is that the constructors take no account of any associated `doseGrid`.
+
+Here are some examples of their use:
+
+```{r}
+.DefaultStoppingAll()
+
+class_name <- "LogisticNormal"
+eval(parse(text = paste0(".Default", class_name, "()")))
+```
+
+# Handling of `NA` or placebo returned as next dose
+
+For consistent handling how the study is stopped and to facilitate analysis of stop reasons in the operation characteristics, the handling of `NA` and placebo returned by `nextBest` methods is changed. In the previous version of `crmPack` stopping for placebo or `NA` returned by a `nextBest` method was handled automatically in the generic `Stopping` method. This is now moved into a new stopping rule `StoppingMissingDose`. As a consequence, the stopping rule `StoppingMissingDose` must be specified for those `nextBest` methods that can return NA, or when placebo is used. Otherwise the simulation may run into an error if the study is not stopped when  `NA` is returned as the next dose. `nextBest` methods that can return `NA` are `NextBestNCRM`, `NextBestNCRMLoss` and `NextBestDualEndpoint`.
+
+# Evaluation of stopping rules at a specific dose
```

Comment:
Can you please double check the levels of the section headers? e.g. this I don't think it should be top level? (but `##` or `###` even)